### PR TITLE
Items in multiple sizes (normal, small and tiny)

### LINKED ISSUE #835: [Enhancement] Item sizes / flavors
**Is your enhancement request related to a problem? Please describe.**
Right now there is only one size for all our items, which determines the size of the items (Padding/Margin etc.)

Also right now the size is dependent on the minimum size which comes from ionic "44" and then it's "8" on each side: 44+8+8 = 60 

https://cookbook.kirby.design/home/showcase/item

In our UI kit there are different sizes, dependent on the use-cases/priorities.

**Describe the solution you'd like**
We would like different sizes which are dependent on the margin and not the min width :)

XS / Tiny = 24+2*4
SM /  = 24+2*8 
Normal / M (Our implemented item right now) = 24+2*16 

**Describe alternatives you've considered**

XS / Tiny : https://zpl.io/aBdMrde
SM : https://zpl.io/VQdnNNm
M / Normal : https://zpl.io/VDoOjNJ

**Additional context**

The 44 min size is made to be sure that the items are clickable (fat-finger), but we have items which are not clickable where we are not reliant on this size. 

Here an example of a screen with a set of items which  have other sizes then our normal: https://zpl.io/ad1Oz8E

### PR DESCRIPTION
**Pull request by MortenGregersen**
Closes #835

### CODE REVIEW

File: src/kirby/components/item/item.component.ts [hunk 1]
```diff
@@ -12,4 +12,6 @@ export class ItemComponent {
   selected;
   @Input()
   selectable: boolean;
+  @Input()
+  size: 'normal' | 'small' | 'tiny' = 'normal';
```
[the remark this answers is not in the document]
Comment (MortenGregersen) [reply src/kirby/components/item/item.component.ts]: I talked with Jakob and Marvin about this.

**Comment by bommariusser** [comment]
normal = medium
tiny = xtra small

should keep same naming.

**Comment by MortenGregersen** [comment]
> normal = medium
> tiny = xtra small
> 
> should keep same naming.

So the sizes should be renamed in Zeplin and we should add a new size (xtra small) to the size directive?

**Comment by hafstad** [comment]
**THIS SHOULD NOT BE MERGED BEFORE FOLLOWING HAS BEEN DECIDED**
Current default size equals to the new small size therefore this pull-req is breaking

**Comment by hafstad** [comment]
Following has been decided with UX:
medium -> 56px min-height with 8px padding-top and padding-bottom
small -> 44px min-height with 8px padding-top and padding-bottom 
extra small -> 32px min-height with 4px padding-top and padding-bottom

**Pull request merged**